mungegithub/mungers/submit-queue.go

```diff
@@ -1305,7 +1324,11 @@ func (sq *SubmitQueue) servePriorityInfo(res http.ResponseWriter, req *http.Requ
       <li>PR with no release milestone will be considered after any PR with a milestone</li>
     </ul>
   </li>
-  <li>PR number</li>
+  <li>First time at which the LGTM label was applied.
```

Comment:
So I should just create my PRs with LGTM and DO_NOT_MERGE and then remove them, right?